kuma/landing/jinja2/landing/react_homepage.html

```diff
@@ -11,7 +11,14 @@
 
   {% if waffle.switch('developer_needs') %}
     {% stylesheet 'banners' %}
+    {% stylesheet 'banner_developer_needs' %}
   {% endif %}
+
+  {% if waffle.flag('mdn_subscriptions') %}
```

Comment:
This will never work. You can't use `waffle.flag` in any Jinja template that is used on the read-only site. It was always be false. The only decent solution, for now, is to have this:
```
{% stylesheet 'banners' %}

{% if waffle.switch('developer_needs') %}
{% stylesheet 'banner_developer_needs' %}
{% endif %}

{% stylesheet 'banner_mdn_subscriptions' %}
```
Yes, it means every single page, authenticated or not, waffled on or not, will have `styles/components/banners/mdn-subscriptions.css` added to it. So, **all pages will become slower when this is landed in master and in production**.

Reply:
So actually, as of this, every page now needs to ship with:

* `http://localhost.org:8000/static/build/styles/banners.92084d28db45.css`  **2.0K**, 701B gzipped
* `http://localhost.org:8000/static/build/styles/mdn-subscriptions.bf637faaf5b0.css` **868B**, 393 gzipped

So it's not a huge increase by any means but it's definitely something to be aware of since we've often talked about the bloat of our CSS be one of the biggest culprits of inferior First Paint Time.